fritz_interview.py

```diff
@@ -0,0 +1,70 @@
+# Lucky Dollar Store Coding Challenge
+import random
+names = ["Eric", "Chris", "Dan", "Jameson"]
+		
+
+def record_customer_purchase(customer):
+     # Fill this in
+     print("- Added new raffle ticket for \"%s\"" % customer)
+     return customer
+
+
+def can_win(customer, pool):
+	if not pool:  
+		return True
+	elif customer in pool.keys():
```

Comment:
You can simplify some of these if statements by using this line:

```
num_raffles_before_eligible = pool.get(customer)
```

Reply:
I simplified this to return true if the num_raffles_before_eligible is 0 or if the key is not found in the dictionary, as suggested above. Otherwise, it returns false.